integration_test/shared/orca-terraform.sh

```diff
@@ -17,58 +16,55 @@ function perform_terraform_command_rds_cluster () {
     -var "cluster_identifier=$bamboo_PREFIX-cumulus-rds-serverless-default-cluster" \
     -var "deletion_protection=false"\
     -var "provision_user_database=false"\
+    -var "tags= { "Deployment" = "$bamboo_PREFIX" }"\
     -var "engine_version=$bamboo_RDS_ENGINE_VERSION" \
     -var "permissions_boundary_arn=arn:aws:iam::$AWS_ACCOUNT_ID:policy/$bamboo_ROLE_BOUNDARY"
 }
 
-function perform_terraform_command_data_persistence () {
+function perform_terraform_command_ecs () {
```

Comment:
Because of the way that repo was set up, it's overly difficult to pull in new versions. I made a new version that doesn't include ORCA code. This way we can deploy the few Cumulus we are dependent on, followed by the ORCA branch that we actually care about.

Reply:
As long as we can perform all the integration tests, then I think we are good.

Reply:
We'll need a separate card to pull in the EC2 instance info and repair any other integration test blockers. This should at least get automated deployment functional and more maintainable.